### DISCUSSION TITLE
My account was hacked

### DISCUSSION THREAD

**Discussion by gapiapja** [community/community#109342] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Not long ago my account was hacked. email, password and username have been changed. what should i do?

**Comment by queenofcorgis** [comment]
Hi @gapiapja ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the community is unable to assist :)

**Comment by asmawi117** [comment]
same issue, try to recovery my hacked account, contact github support and got no response in 35 days.
good luck 😥

**Reply by digidez-dev** [reply]
same issue :((

**Reply by iamPedram1** [reply]
same here its been a month and no respond from github!

**Comment by SebastienXia** [comment]
## Hello, @gapiapja 

Your account is your email in general or your github account ? because its not the sale thing :/
